src/rapids_dependency_file_generator/schema.json

```diff
@@ -0,0 +1,161 @@
+ {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://raw.githubusercontent.com/rapidsai/dependency-file-generator/v1.0.0/src/rapids_dependency_file_generator/schema.json",
```

Comment:
@ajschmidt8 I'm not sure how best to square including a version number in this id with our use of `semantic-release`. See https://github.com/vyasr/dependency-file-generator/pull/1#discussion_r1061750555 for a longer discussion. Could you comment on what you think is the best path forward here?

Reply:
@vyasr. we can do that with https://github.com/semantic-release/git.

We'll have to figure out how to configure it, but it will commit "release assets" back to the repository (with a `[skip ci]` tag) after a release has occurred.

In this case, `schema.json` would be categorized as a "release asset" since it depends on the release version.

Reply:
OK got it. Is that something that you want to have done before this PR, or are you OK with having this entry be slightly out of date until we implement that afterwards? I don't want to balloon this PR by adding that change here.

Reply:
We can add that in the future.

I can work on that if you don't mind opening up an issue for me.

Reply:
https://github.com/rapidsai/dependency-file-generator/issues/30